dub.sdl

```diff
@@ -7,6 +7,8 @@ license "MIT"
 
 targetPath "bin"
 
+dependency "dyaml" version=">=0.8.6"
```

Comment:
I think we might not be allowed to add dependencies to dub, as per this summary from 2013 of a statement that Walter is supposed to have made: https://forum.rejectedsoftware.com/groups/rejectedsoftware.dub/post/51

Reply:
And it doesn't build because bootstrapping :)
dyaml is already in dlang-community. It could also be vendored. I think that's a minor detail which we can solve at the polishing phase, do you agree ?

Reply:
yeah it's fine for me, don't know what the current stance from those releasing & distributing is.

If we add a dependency it's probably also relevant for package maintainers of various linux distros?